src/test/java/EncodeTest.java

```diff
@@ -0,0 +1,70 @@
+import encoder.Encoder;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+/** Validate that a .txt file can be properly encoded into a protobuf file. */
+public final class EncodeTest {
+  private static String inputFile;
+  private static String outputFile;
+  private static InputStream input;
+  private static OutputStream output;
+
+  @Before
+  public void setUp() {
+    inputFile = "testInput.txt";
+    outputFile = "testOutput.sstable";
+    try {
+      input = new FileInputStream(inputFile);
+    } catch (FileNotFoundException e) {
+      System.out.format("Unable to open the input file with name %s", inputFile);
```

Comment:
can the error message be more descriptive? like is it unable to print bc the given files the wrong format or bc the encoder is broken?

Reply:
I'm not totally sure what would be more descriptive, a filenotfoundexception occurs when the file cannot be found usually because it doesn't exist in the current diretory?

Reply:
oh ok then. i thought bout FileNotFoundException error messages were the same, but i notice that ones for input and the others for output